aravisGigEApp/src/aravisCamera.cpp

```diff
@@ -203,6 +225,16 @@ static void newBufferCallback (ArvStream *stream, aravisCamera *pPvt) {
         // printf as pPvt->pasynUserSelf for asynPrint is protected
         printf("Bad frame status: %d size: %lu\n", buffer_status, size);
```

Comment:
Yes, that lines not in my master, but got left in this topic branch.
I'm also removing the call to arv_buffer_get_data() above that fetches size, as it's always
just the preallocated size so isn't helpful.